src/ImageSharp/Memory/Allocators/Internals/SharedArrayPoolBuffer{T}.cs

```diff
@@ -22,7 +21,7 @@ public SharedArrayPoolBuffer(int lengthInElements)
         this.lifetimeGuard = new LifetimeGuard(this.Array);
     }
 
-    public byte[] Array { get; private set; }
+    public byte[]? Array { get; private set; }
```

Comment:
What are uses of this property outside of the class? Can make it private instead of making it nullable?

Reply:
It is used in MemoryGroupSpanCache:30

I can make it non nullable. But then I have to remove the null check in the dispose and the explicit setting it to null in the dispose method

Reply:
In that case `byte[]? Array` seems ok.